packages/amplify-frontend-android/lib/frontend-config-creator.js

```diff
@@ -1,6 +1,48 @@
 const constants = require('./constants');
 const path = require('path');
 const fs = require('fs-extra');
+const graphQLConfig = require('graphql-config');
+
+const FILE_EXTENSION_MAP = {
+  javascript: 'js',
+  graphql: 'graphql',
+  flow: 'js',
+  typescript: 'ts',
+  angular: 'graphql',
+};
+
+const fileNames = ['queries', 'mutations', 'subscriptions'];
+
+function deleteAmplifyConfig(context) {
+  const { amplify } = context;
+  const projectPath = context.exeInfo ? context.exeInfo.localEnvInfo.projectPath : amplify.getEnvInfo().projectPath;
+  const projectConfig = context.exeInfo ? context.exeInfo.projectConfig[constants.Label] : amplify.getProjectConfig()[constants.Label];
+  const frontendConfig = projectConfig.config;
+  const srcDirPath = path.join(projectPath, frontendConfig.ResDir, 'raw');
+  const targetFilePath = path.join(srcDirPath, constants.amplifyConfigFilename);
```

Comment:
I wonder if we could move this to a helper function since its a copy past from the function below as is

Reply:
Where do you think the helper function should go?